vision/src/autogluon/vision/object_detection/object_detection.py

```diff
@@ -13,32 +14,36 @@ class ObjectDetector(object):
 
     Parameters
     ----------
-    log_dir : str
-        The directory for saving logs, by default using `pwd`: the current working directory.
-
+    path : str, default = None
+        The directory for saving logs or intermediate data. If unspecified, will create a sub-directory under
+        current working directory.
+    verbosity : int, default = 2
+        Verbosity levels range from 0 to 4 and control how much information is printed. 
+        Higher levels correspond to more detailed print statements (you can set verbosity = 0 to suppress warnings). 
+        If using logging, you can alternatively control amount of information printed via logger.setLevel(L), 
+        where L ranges from 0 to 50 (Note: higher values of L correspond to fewer print statements, opposite of verbosity levels)
     """
     # Dataset is a subclass of `pd.DataFrame`, with `image` and `bbox` columns.
     Dataset = _ObjectDetection.Dataset
 
-    def __init__(self, log_dir=None):
-        self._log_dir = log_dir
+    def __init__(self, path=None, verbosity=2):
```

Comment:
will we ever have a reason to have different eval_metric or problem_type for this?

Reply:
seems important to allow for other eval_metric in the future